install/index.md

```diff
@@ -283,7 +289,7 @@ Windows users can now tap into GPU accelerated data science on their local machi
 
 ### **WSL2 Additional Prerequisites**
 
-<i class="fas fa-desktop text-white"></i> **OS:** Windows 11 with Ubuntu 22.04 instance for WSL2. <br/>
+<i class="fas fa-desktop text-white"></i> **OS:** Windows 11 with a WSL2 installation of Ubuntu (minimum version 20.04). <br/>
```

Comment:
Looks like this was Ubuntu 22.04 and then became 20.04. Was this meant to still be 22.04?

```suggestion
<i class="fas fa-desktop text-white"></i> **OS:** Windows 11 with a WSL2 installation of Ubuntu (minimum version 22.04). <br/>
```

Reply:
This should be Ubuntu 20.04. It was incorrect before.